tools/cd_scripts/install_test.sh

```diff
@@ -32,42 +32,53 @@ EOF
 #details.txt file contains the release version and commit hash of the current release.
 gsutil cp  gs://gcsfuse-release-packages/version-detail/details.txt .
```

Comment:
@ashmeenkaur @sethiay please check this file. This had a lot of conflicts on merging between the following commits.

* https://github.com/GoogleCloudPlatform/gcsfuse/commit/70ed0ef3f47d605de3354b3652b77617b3bc26ea
* https://github.com/GoogleCloudPlatform/gcsfuse/commit/a510793b68715877c95d77f1c06bd406a638b292

Reply:
@Tulsishah  for change in arm-related changes.

Reply:
Please go with @ashmeenkaur latest changes. I checked looks good to me.

Reply:
Please accept the changes from release branch.

Reply:
> Please accept the changes from release branch.

Done.

Also, I realized where most of the conflicts were from.
1. master: https://github.com/GoogleCloudPlatform/gcsfuse/commit/70ed0ef3f47d605de3354b3652b77617b3bc26ea
2. dec_2023_release: https://github.com/GoogleCloudPlatform/gcsfuse/commit/dc6aecaed2fbb326b6415f165cb5b4c173465e63